test/content-type.js

```diff
@@ -0,0 +1,157 @@
+  ['https://whatwg.org/demos/date-01/test-source', 'text/plain; charset=utf-8'],
+  ['https://whatwg.org/demos/multiform-01/order', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/demos/multiform-01/order-source', 'text/plain; charset=utf-8'],
+  ['https://whatwg.org/demos/multiform-01/postcode-lookup', 'application/xml'],
+  ['https://whatwg.org/demos/multiform-01/postcode-lookup-source', 'text/plain; charset=utf-8'],
+  ['https://whatwg.org/demos/multiform-01/product-lookup', 'application/xml'],
+  ['https://whatwg.org/demos/multiform-01/product-lookup-source', 'text/plain; charset=utf-8'],
+  ['https://whatwg.org/demos/repeat-01/test', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/demos/repeat-01/test-source', 'text/plain; charset=utf-8'],
+  ['https://whatwg.org/faq', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/mailing-list', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/news/future-of-html', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/news/start', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/news/web-forms-call-for-comments-1', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/news/web-forms-call-for-comments-2', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/news/web-forms-call-for-comments-3', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/news/web-forms-call-for-comments-4', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/specs/web-apps/2006-01-01/diff-2005-09-01', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/specs/web-forms/2004-06-27-call-for-comments/sample-datetime-ui-1', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-06-27-call-for-comments/sample-datetime-ui-2', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-06-27-call-for-comments/sample-datetime-ui-3', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-06-27-call-for-comments/sample-multiple-editable-ui', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-06-27-call-for-comments/xforms-implementation-diagram', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-12-10-call-for-comments/diff-2004-06-27-call-for-comments', 'text/html; charset=utf-8'],
+  ['https://whatwg.org/specs/web-forms/2004-12-10-call-for-comments/sample-autocompletion-ui-1', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-12-10-call-for-comments/sample-autocompletion-ui-2', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-12-10-call-for-comments/sample-datetime-ui-1', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-12-10-call-for-comments/sample-datetime-ui-2', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-12-10-call-for-comments/sample-datetime-ui-3', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-12-10-call-for-comments/sample-multiple-editable-ui', 'image/png'],
+  ['https://whatwg.org/specs/web-forms/2004-12-10-call-for-comments/xforms-implementation-diagram', 'image/png'],
+  // much more of the same...
+  /*
```

Comment:
Why keep this commented-out block?

Reply:
I wanted to make the tests run a bit faster by skipping some bits. And easier testing of everything when doing the nginx setup. But between deleting and including, I think I'll just run all of the tests, and maybe we can make this smaller later using http://nginx.org/en/docs/http/ngx_http_core_module.html#try_files.